openmdao/components/meta_model_structured_comp.py

```diff
@@ -368,8 +361,6 @@ def _evaluate_splines(self, data_values, xi, indices, interpolator, method,
         compute_gradients : bool, optional
             If a spline interpolation method is chosen, this determines whether gradient
             calculations should be made and cached. Default is True.
-        first_dim_gradient : bool, optional
```

Comment:
I removed it mainly because it there weren't any options on the component that let you use it. Also, it was completely untested, so I wasn't sure it was even right (and it seemed to me the values it poked into `all_gradients` just got overwritten in the subsequent lines anyway.)